teachopencadd/talktorials/T014_binding_site_detection/talktorial.ipynb

```diff
@@ -0,0 +1,2116 @@
+{
```

Comment:
<p>Remove</p><p><code># Get URL for all PDB files</code></p><p>comment, since it could be ccp2 or pdb.</p><p> </p><p>Could you add a few comments on the rational behind these conditions?</p><p> </p><p><code>&nbsp;&nbsp;&nbsp;for pocket_file in pocket_files:</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;if file_type == 'pdb':</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;if f'{best_pocket}_res' in pocket_file:</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;result.append(pocket_file)</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;elif file_type == 'ccp4':</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;if f'{best_pocket}_gpsAll' in pocket_file:</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;result.append(pocket_file)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
<p>Done.</p>